src/lib.rs

```diff
@@ -56,6 +56,24 @@ where
     Err(())
 }
 
+#[cfg(target_env = "gnu")]
+mod glibc {
+    use std::path::Path;
+
+    // Attempt to compile a c program that links to strlcpy from the std
+    // library to determine whether glibc packages it.
```

Comment:
Actually to be more correct, `cc` crate never links, only compiles (it uses `-c` flag).  The reason that this works is because gcc can not find the function declaration and generates a warning. Then warnings become errors because of `warnings_into_errors(true)`.

Reply:
Thanks for the clarification. I'll update the comment.